### PR TITLE
Added missing tags/teams

### PR DESCRIPTION
**Pull request by hannah38**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by zc-hmcts — CHANGES_REQUESTED** [review]

File: team-config.yml [hunk 1]
```diff
@@ -211,4 +211,94 @@ shared-image-gallery:
...
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "caps"
+    businessArea: "Cross-Cutting"
+
+pcol:
+  team: "pcol"
```
[the remark this answers is not in the document]
Comment (darren221980) [reply team-config.yml:227]: Not sure what CAPS or PCOL actually stand for as the tagging dictionary doesn't state this.

I've checked the repo that the terraform code resides in for those services and the two contacts mentioned in the readme don't appear to be around. As a result I've asked the most recent person who updated that repo if they can advise and once they do I shall update this pr.

[the remark this answers is not in the document]
Comment (darren221980) [reply team-config.yml]: @zc-hmcts team names updated.

File: team-config.yml [hunk 2]
```diff
@@ -211,4 +211,84 @@ shared-image-gallery:
...
+  namespace: ""
+  slack:
+    contact_channel: ""
+    build_notices_channel: ""
+  tags:
+    application: "heritage-secure-data-transfer"
+    businessArea: "Cross-Cutting"
+
+template-management:
+  team: "template-management"
```
Comment (Jenniferstrej) [inline team-config.yml:287]: Just checking why this and darts-idam don't follow the same pattern as a full team name?

Comment (darren221980) [reply team-config.yml:287]: @Jenniferstrej updated Darts-idam as found this in Confluence.

Have left template-management as is, as that is what it should be according to information within Confluence spaces.

**Review by Jenniferstrej — APPROVED** [review]

**Pull request merged**